flow2ml/Data_Augumentation.py

```diff
@@ -109,4 +109,54 @@ def applyShear(self,classPath):
             # saving the image by
             plt.imsave(classPath+"/ShearedImages/Sheared"+image, cv2.cvtColor(Sheared, cv2.COLOR_RGB2BGR))
           except Exception as e:
-            print(f"Shearing operation failed due to {e}")
\ No newline at end of file
+            print(f"Shearing operation failed due to {e}")
+
+  def applyInvert(self,classPath):
+    '''
+      Applies invert augmentation to all the images in the given folder. It negates the images by reversing the pixel value.
+      Args : 
+        classPath : (string) directory containing images for a particular class.
+    '''
+    try:
+      os.mkdir(classPath+"/InvertedImages")    
+    except:
+      raise Exception("Unable to create directory for inverted images.")
+    
+    for image in list(os.listdir(classPath)):
+      # Read image
+      img = cv2.imread(classPath+"/"+image)
+      if (self.operations['invert']==True):
+        if img is not None:
+          try:
+            # applies Invert augmentation to the image.
+            Inverted=abs(255-img)
+            # saving the image by
+            plt.imsave(classPath+"/InvertedImages/Inverted"+image, Inverted)
+          except Exception as e:
+            print(f"Inverting operation failed due to {e}")
+
+  def HistogramEqualization(self,classPath):
```

Comment:
Why do we have code for Histogram Equalization here?

Reply:
> Why do we have code for Histogram Equalization here?

you  only have told na?